website/docs/docs/cloud/manage-access/external-oauth.md

```diff
@@ -23,7 +20,7 @@ dbt Cloud Enterprise supports [external OAuth authentication](https://docs.snow
 
 ## Getting started
```

Comment:
DO we need to link to the snowflake doc in the above paragraph?

Reply:
Good call!